nextflow_schema.json

```diff
@@ -58,6 +47,60 @@
                 }
             }
         },
+        "alignment_parameters": {
+            "title": "Alignment parameters",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "aligner": {
+                    "type": "string",
+                    "description": "Aligner program to use",
+                    "default": "minimap2",
+                    "fa_icon": "fas fa-align-justify",
+                    "enum": ["minimap2", "bwa", "bowtie2"]
+                }
+            }
+        },
+        "screening_parameters": {
+            "title": "Screening parameters",
+            "type": "object",
+            "description": "Parameters used for functional genomic screenings",
+            "default": "",
+            "properties": {
+                "mle_design_matrix": {
+                    "type": "string",
+                    "description": "Design matrix used for MAGeCK MLE to call essential genes under multiple conditions while considering sgRNA knockout efficiency"
+                },
+                "rra_contrasts": {
+                    "type": "string",
+                    "description": "file comma separated on the conditions needing to be compared. The first one will be the reference (control)"
```

Comment:
```suggestion
                    "description": "Comma-separated file with the conditions to be compared. The first one will be the reference (control)"
```

Reply:
I would suggest to start all descriptions uppercase and have a look at how the parameter schema looks like on the website to make it nicely readable for the user.